Backend/starter_project_zsilentcoders/BlogApp3/BlogApp.Application/Features/Reviews/DTOs/Validators/DeleteReviewValidator.cs

```diff
@@ -0,0 +1,23 @@
+﻿using BlogApp.Application.Contracts.Persistence;
+using FluentValidation;
+using MediatR;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BlogApp.Application.Features.Reviews.DTOs.Validators
+{
+    public class DeleteReviewValidator: AbstractValidator<int>
+    {
+
+        public DeleteReviewValidator(IUnitOfWork _unitOfWork)
+        {
+            RuleFor(id=>id)
+                .MustAsync(async (id, token) => await _unitOfWork.ReviewRepository.Exists(id))
```

Comment:
good job to check if the review exists or not first. Do the same for updating the review